apis/meta/v1alpha1/component.go

```diff
@@ -31,24 +30,6 @@ const (
 
 var ErrUnsupportedComponent = errors.New("component not supported")
 
-func GetLargestComponentNameSize() int {
-	max := len(Component(math.MaxInt64).String())
-
-	size := len(Component(math.MinInt64).String())
-	if size > max {
-		max = size
-	}
-
-	for i := 0; i < int(componentCount); i++ {
-		size := len(Component(i).String())
-		if size > max {
-			max = size
-		}
-	}
-
-	return max
-}
```

Comment:
FYI This was used by webhooks to validate `.metadata.name` of custom resources.
Since operator adds suffixes to the name of owned resources, it can reach the native Kubernetes limits: <https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names>.
This validation methods check that child resources will be valid.

Reply:
got it